### ISSUE TITLE
(suggestion) incremental power scrolls

### ISSUE THREAD

**Issue by EstelMatiazi** [Secrets-of-Sosaria/World#130]
After some discussion on discord, a user brought an interesting suggestion:

What if we had powerscrolls that add +5 to skill cap on a given skill instead of setting it to a fixed amount?

I enjoyed this idea quite a bit, mostly because power scrolls are not heavily used these days due to gear with +skill overwriting skill cap (which, in my opinion, should not be a thing, but that's a subject for another Issue), and having more options could bring more interest to the system. 

What I have in mind for this system is that these +5 scrolls should be one of the rare options available as loot drops from enemies that have more than 10k fame and karma, getting slightly more common into max fame/karma enemies, with powerful endgame bosses having the possibility of dropping more than one. The skill in question will always be random, and overtime characters will build up scrolls for their alts and for trading.

**Comment by Kynick** [comment]
Would this effect total skill points at all?

**Comment by EstelMatiazi** [comment]
No, total skill points remain the same, just individual skill limits are affected by these items.

**Comment by mz1193** [comment]
This is an intriguing idea. Do you have a hard cap in mind, or is the idea that these would be so rare that implementing a hard cap would be meaningless?

**Comment by EstelMatiazi** [comment]
Cap stays at 125 per skill, so you would need 5 scrolls per skill to get it maxxed out.

**Comment by mz1193** [comment]
> Cap stays at 125 per skill, so you would need 5 scrolls per skill to get it maxxed out.

Got it, thanks! I think this idea is a good one :)
